x/feemarket/keeper/params.go

```diff
@@ -71,7 +71,11 @@ func (k Keeper) GetBaseFee(ctx sdk.Context) *big.Int {
 // SetBaseFee set's the base fee in the store
 func (k Keeper) SetBaseFee(ctx sdk.Context, baseFee *big.Int) {
 	params := k.GetParams(ctx)
-	params.BaseFee = sdkmath.NewIntFromBigInt(baseFee)
+	if baseFee.BitLen() > sdkmath.MaxBitLen {
+		params.BaseFee = k.maxGas
```

Comment:
what's the point of this, this doesn't prevent overflows in other calculations, right?

Reply:
when calculating the base fee, math/big is used instead of the sdk library which shouldn't typically overflow. at least in the feemarket module, this method is only called in begin block right after the calculation. so this fix would be more capping the base fee to the maximum value supported by `sdkmath.Int` and would prevent any overflows coming from the feemarket module.

Reply:
I see, you want a different way to convert *big.Int to sdkmath.Int